<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="altaCmd">
       <attribute name="title">
        <string>Alta command</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="3">
           <widget class="QLCDNumber" name="temperatureLCD"/>
          </item>
          <item row="5" column="2">
           <widget class="QSpinBox" name="unloadLengthSpin">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QSpinBox" name="loadLengthSpin">
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="stepType">
             <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="backPointButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="beginCalibButton">
            <property name="text">
             <string>Initialize</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="unloadButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Unload Filament</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="frontLeftButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Front Left</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="cooldownExtruderButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cooldown Extruder</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Extruder Temperature:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>STOP !!!</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="loadButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Load Filament</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="dispenserButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Dispenser point</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="frontRightButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Front Right</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="serialManager">
       <attribute name="title">
        <string>Serial Manager</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QWidget" name="managerContainer" native="true">
          <layout class="QGridLayout" name="gridLayout_4"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logTab">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>frontLeftButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>frontRightButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>dispenserButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>cooldownExtruderButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>loadButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>unloadButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>598</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>beginCalibButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>backPointButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
